examples/basic/main.tf

```diff
@@ -0,0 +1,18 @@
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+  name      = "${var.name}"
+}
```

Comment:
add
cidr_block = "10.0.0.0/16"

Reply:
make `cidr_block` a var and use for both `vpc` and `subnets`

Reply:
look at changes plz)